### ISSUE TITLE
Version 11.5 and 11.4 removed Ring Devices from HomeKit

### ISSUE THREAD

**Issue by RadDip** [dgreif/ring#1098]
# Bug Report

### Describe the Bug
Upgrading to both 11.4 and then 11.5 caused all ring devices to be removed from HomeKit.  Had to downgrade to version 11.3.1 and then create a new refresh token

### To Reproduce
 - Start Homebridge
 - Attempt to ...
 - See error

### Expected behavior
All ring devices would remain intact and operation in HomeKit

### Homebridge Ring Config
Post homebridge-ring platform config  _without sensitive information_
```

HIGHLIGHT THIS TEXT AND POST CONFIG HERE
"platform": "Ring",
    "refreshToken": "xxxxxx",
    "sendCameraMotionNotificationsToTv": true,
    "sendDoorbellMotionNotificationsToTv": true,
    "showPanicButtons": false,
    "unbridgeCameras": false

```

### Environment
Hoobs Version 4.2.8
Node: 16.18.1
Homebridge Version 1.5.0
System version: 11

**Comment by ED7779** [comment]
Same, after upgrading to 11.5 I lost all the devices.

**Comment by dgreif** [comment]
> Hoobs Version 4.2.8

@RadDip Just to confirm, are you using HOOBS? @ED7779 same question for you

**Comment by cantchooseaname8** [comment]
I lost all devices too on the latest update. I downgraded the plugin version through HOOBS and the devices came back. However, they all defaulted to the same room within HomeKit instead of going back to where they originally were. The devices were also removed from all of the automations and needed to be set up again. Not sure if this helps at all but just wanted to pass along what I noticed.

**Comment by mcooper333** [comment]
Same issue here. Confirming that I have HOOBs.

**Comment by dgreif** [comment]
Can you try v11.5.1?

**Comment by mcooper333** [comment]
Works great! Thanks.Also, does this update solve the issue with Apple TV’s? After a few days, I noticed it drops the video connection with TVOS but still shows video feed through the home iOS app. I’m usually able to get the video feed back once I unplug and plug the Apple TV back into the power source. Assuming this is more of an apple issue?  On Nov 14, 2022, at 12:50 PM, Dusty Greif ***@***.***> wrote:﻿
Can you try v11.5.1?

**Comment by dgreif** [comment]
@mcooper333 that does sound like more of an Apple issue, sorry!

Closing as this appears to be resolved by 11.5.1 🎉
